packages/ui5-application-writer/templates/optional/typescript/tsconfig.json

```diff
@@ -13,7 +13,11 @@
     "rootDir": "webapp",
     "outDir": "./dist",
     "baseUrl": "./",
-    "paths": {
+    "paths": {<% if (app.id.includes('.')) { %>
```

Comment:
To simplify the code, I would remove the if, and just always replace `.` with `/`. There is no harm in running on a string without dots.